src/Evolution/DiscontinuousGalerkin/Initialization/Mortars.cpp

```diff
@@ -61,7 +61,7 @@ mortars_apply_impl(const std::vector<std::array<size_t, Dim>>& initial_extents,
     normal_covector_quantities[direction] = std::nullopt;
     for (const auto& neighbor : neighbors) {
       const auto mortar_id = std::make_pair(direction, neighbor);
-      mortar_data[mortar_id];  // Default initialize data
+      mortar_data[mortar_id] = MortarData<Dim>{1};
```

Comment:
`mortar_data.emplace(mortar_id, MortarData<Dim>{1});`  (Although I think you made this the default in the last commit.)